addons/crm_claim/migrations/9.0.1.0/pre-migration.py

```diff
@@ -17,7 +17,13 @@
 ]
 
 
-@openupgrade.migrate()
-def migrate(cr, version):
-    openupgrade.rename_columns(cr, column_renames)
-    openupgrade.rename_tables(cr, table_renames)
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    openupgrade.rename_columns(env.cr, column_renames)
+    openupgrade.rename_tables(env.cr, table_renames)
+    # Portal_claim is merged here and It is needed rename xml-id
```

Comment:
This is not needed anymore making the renaming in the other place. Please restore it as it was.